src/Error_Logging/error_logging.py

```
@@ -0,0 +1,51 @@
+    
+        action = data.action
+        outcome = data.outcome
+        info = data.info
+        duration_ms = data.duration_ms
+        error_code = data.error_code
+
+        log_action(action=action, outcome=outcome, info=info, duration_ms=duration_ms, code=error_code)
+        
+        return {"success": True, "message": "Successfully logged error"}
+
+    except Exception as error:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)